### PR TITLE
fix: add noop message to relay to triger extension of topic

### LINKED ISSUE #92: Send noop messages as hotfix to lengthen the topic in relay

### PR DESCRIPTION
**Pull request by Rakowskiii**
# Description

Adding noop message in subscribe flow to trigger topping mapping extension in relay.

Resolves #92 

## How Has This Been Tested?

Tests passing.

## Due Diligence

* [ ] Breaking change
* [ ] Requires a documentation update
* [ ] Requires a e2e/integration test update

### CODE REVIEW

File: src/websocket_service/handlers/push_subscribe.rs [hunk 1]
```diff
@@ -95,14 +95,26 @@ pub async fn handle(
...
 
+    // This noop message is making relay aware that this topics TTL should be
+    // extended
+    info!("[{request_id}] Sending settle message");
+    client
+        .publish(
+            push_topic.into(),
+            "",
+            4050,
+            Duration::from_secs(86400),
```
Comment (arein) [inline src/websocket_service/handlers/push_subscribe.rs:113]: can be lower

File: src/websocket_service/handlers/push_subscribe.rs [hunk 2]
```diff
@@ -84,7 +84,7 @@ pub async fn handle(
             response_topic.into(),
             base64_notification,
             4007,
-            Duration::from_secs(86400),
+            Duration::from_secs(300),
```
Comment (arein) [inline src/websocket_service/handlers/push_subscribe.rs:87]: curious: why?

File: src/websocket_service/handlers/push_subscribe.rs [hunk 3]
```diff
@@ -95,14 +95,26 @@ pub async fn handle(
...
-        &client_data.id,
-        &sha256::digest(&*hex::decode(&push_key)?),
-        &project_data.id,
-        &client_data.scope
+        &client_data.id, &push_topic, &project_data.id, &client_data.scope
     );
 
+    // This noop message is making relay aware that this topics TTL should be
+    // extended
+    info!("[{request_id}] Sending settle message");
```
Comment (arein) [inline src/websocket_service/handlers/push_subscribe.rs:107]: ideally: can you add the topic here?

**Review by arein — APPROVED** [review]
Looks good small changes

File: src/websocket_service/handlers/push_subscribe.rs (hunk 3 above)
Comment (Rakowskiii) [reply src/websocket_service/handlers/push_subscribe.rs:107]: Sure

File: src/websocket_service/handlers/push_subscribe.rs (hunk 2 above)
Comment (Rakowskiii) [reply src/websocket_service/handlers/push_subscribe.rs]: Good catch, I wanted to change TTL for the NoOp not this

File: src/websocket_service/handlers/push_subscribe.rs [hunk 4]
```diff
@@ -95,14 +95,26 @@ pub async fn handle(
...
+    client
+        .publish(
+            push_topic.into(),
```
[the remark this answers is not in the document]
Comment (Rakowskiii) [reply src/websocket_service/handlers/push_subscribe.rs]: We don't care about extending the response_topic, it's disposable. We want to ensure push one stays alive

**Review by chris13524 — APPROVED** [review]

**Pull request merged**